src/MonoTorrent/MonoTorrent.Streaming/StreamProvider.cs

```diff
@@ -159,15 +159,18 @@ public Task<IUriStream> CreateHttpStreamAsync (ITorrentFileInfo file, Cancellati
         public Task<IUriStream> CreateHttpStreamAsync (ITorrentFileInfo file, bool prebuffer)
             => CreateHttpStreamAsync (file, prebuffer, CancellationToken.None);
 
+
         /// <summary>
         /// Creates a <see cref="Stream"/> which can be used to access the given <see cref="TorrentFile"/>
         /// while it is downloading. This stream is seekable and readable. This stream must be disposed
         /// before another stream can be created.
         /// </summary>
         /// <param name="file">The file to open</param>
         /// <param name="prebuffer">True if the first and last piece should be downloaded before the Stream is created.</param>
+        /// <param name="serverUri">Points to another server requesting stream's content without CORS Policy Issue.</param>
```

Comment:
There's a couple of extra empty lines which would be nice to remove (if you're comfortable making those changes). Otherwise it would be good to remove just this one as the parameter doesn't exist in this version of the patch!

Reply:
Oopsies. I forgot about that. Alright, removing those lines since it's no longer needed. :)